src/ApproxConv.jl

```diff
@@ -168,6 +174,20 @@ function calcVariableCovarianceBasic(ptsArr::Vector{Vector{Float64}})
   return msst_
 end
 
+function calcVariableCovarianceBasic(M::AbstractManifold, ptsArr::Vector{P}) where P
+  #TODO double check the maths,. it looks like its working at least for groups
+  μ = mean(M, ptsArr)
+  Xcs = vee.(Ref(M), Ref(μ), log.(Ref(M), Ref(μ), ptsArr))
+  Σ = mean(Xcs .* adjoint.(Xcs))
+  @debug "calcVariableCovarianceBasic" μ
+  @debug "calcVariableCovarianceBasic" Σ
+  # TODO don't know what to do here so keeping as before, #FIXME it will break
+  # a change between this and previous is that a full covariance matrix is returned
+  msst = Σ
+  msst_ = 0 < sum(1e-10 .< msst) ? maximum(msst) : 1.0
+  return msst_
+end
+
```

Comment:
@dehann, have a look please

Reply:
msst_ is probably part of initialization when trivial case errors occur in this calculation.  I was expecting the first mean for mu, the coordinates on `Xc` look right (around mu).  The adjoint is somewhat unfamiliar to me in this context, must just make sure that the communativity is right, e.g. hypothetically:
```
# good
aRc = adjoint(bRa)*bRc

# vs bad
bRa*adjoint(bRc)
```
since the latter would give incorrect answers.  I'm not sure and would need to dig more to learn more.

Reply:
Xcs is in coordinates, and transpose is just used to build the matrix to be used for the covariance

Reply:
See #1280